tools/filesystem/file.go

```diff
@@ -87,6 +106,27 @@ func (r *PathReader) Open() (io.ReadSeekCloser, error) {
 
 // -------------------------------------------------------------------
 
+var _ FileReader = (*bytesReader)(nil)
+
+type bytesReader struct {
```

Comment:
Similar to `PathReader` and `MultipartReader` reader, let's export this because we may use it as embedded struct or as a return type in some other code.

The `bytesReadSeekCloser` can remain unexported.